### PR TITLE
chore(demo-playwright): test for `checkbox`

### PR DESCRIPTION
**Pull request by AtharvChandratre**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Addresses https://github.com/taiga-family/taiga-ui/issues/4912

## What is the new behaviour?
This PR adds an e2e test (with playwright) for `checkbox`

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
+    [`m`, `l`].forEach(size => {
+        test(`size=${size}`, async ({page}) => {
+            await tuiGoto(page, `components/checkbox/API?size=${size}`);
+            const example = page.locator(`#demo-content`);
+
+            await expect(example).toHaveScreenshot(`checkbox-size-${size}.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:10]:
```suggestion
            const {apiPageExample} = new TuiDocumentationPagePO(page);

            await expect(apiPageExample).toHaveScreenshot(`checkbox-size-${size}.png`);
```

File: projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:4]:
```suggestion
test.describe(`Checkbox`, () => {
```

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts (hunk 1 above)
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:10]: Too wide screenshot

Screenshot 2023-11-20 at 13 37 01

Comment (AtharvChandratre) [reply projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:10]: Screenshot 2023-11-20 at 1 03 42 PM

The Cypress screenshot size was 1758x416, so I thought that was how wide we wanted the screenshot. Do you have any advice on what the size should be? I can try to force that size of the screenshot.

Comment (AtharvChandratre) [reply projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:10]: Addressed

File: projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts (hunk 2 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:4]: Oh sorry for that 😅, addressed this comment

File: projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts]: You can use `await page.setViewportSize({width, height})` for this test.
For example, you can set it as `300*300`.

See the example:
https://github.com/taiga-family/taiga-ui/blob/1e624c0f36e68d02abc642b89e606402f57225cc/projects/demo-playwright/tests/core/breakpoint-service/breakpoint-service.spec.ts#L11

Comment (AtharvChandratre) [reply projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts]: Thank you, made this change

File: projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import {expect, test} from '@playwright/test';
+
+test.describe(`Checkbox`, () => {
+    [`m`, `l`].forEach(size => {
+        test(`size=${size}`, async ({page}) => {
+            await page.setViewportSize({width: 300, height: 300});
+            await tuiGoto(page, `components/checkbox/API?size=${size}`);
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+            await expect(apiPageExample).toHaveScreenshot(`checkbox-size-${size}.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/checkbox/checkbox.spec.ts:11]: ✅ 

checkbox-size-l-chromium-darwin

___
✅ 
checkbox-size-m-chromium-darwin

**Review by nsbarsukov — APPROVED** [review]

**Comment by nsbarsukov** [comment]
@AtharvChandratre 
Thank you ❤️

**Pull request merged**